src/js/tabs/osd.js

```diff
@@ -774,6 +774,14 @@ OSD.constants = {
         draw_order: 320,
         positionable: true,
         preview: 'AG'
+    },
+    G_FORCE: {
+        name: 'G_FORCE',
+        desc: 'osdDescGForce',
+        default_position: -1,
+        draw_order: 11,
```

Comment:
Is better to use 15 as draw order, to let add elements before and after if needed.